shared/src/validation-schemas/podcast/create-podcast.validation-schema.ts

```diff
@@ -0,0 +1,10 @@
+import * as Joi from 'joi';
+import { PodcastCreatePayloadKey } from '~/common/enums/enums';
+import { PodcastCreatePayload } from '~/common/types/types';
+
+const createPodcastSchema = Joi.object<PodcastCreatePayload>({
+  [PodcastCreatePayloadKey.NAME]: Joi.string().min(1).max(20).required(),
```

Comment:
Create enum for:
rules - podcastValidationRule
messages: podcastMessage
in the `shared/src/common/enums/validation/`